src/disasm.cpp

```diff
@@ -271,7 +271,14 @@ static void print_source_line(raw_ostream &stream, DebugLoc Loc)
 }
 #endif
 
-extern "C"
+extern "C" {
+JL_DLLEXPORT void jl_init_disasm() {
```

Comment:
are these functions expensive? it would be perhaps be easier to just call this from jl_codegen_init once and not think about again